applets/wncklet/window-list.c

```diff
@@ -404,9 +410,10 @@ static gboolean applet_enter_notify_event (WnckTasklist *tl, GList *wnck_windows
 	tasklist->preview = gtk_window_new (GTK_WINDOW_POPUP);
 
 	gtk_widget_set_app_paintable (tasklist->preview, TRUE);
-	gtk_window_set_resizable (GTK_WINDOW (tasklist->preview), TRUE);
+	gtk_window_set_default_size (GTK_WINDOW (tasklist->preview), thumbnail_width/thumbnail_scale, thumbnail_height/thumbnail_scale);
+	gtk_window_set_resizable (GTK_WINDOW (tasklist->preview), FALSE);
```

Comment:
For some reason setting this as `FALSE` prevents large windows from showing up as thumbnails.

Reply:
@vkareh done